### PR TITLE
driver/ata8520e: minor doc fixes

### PR DESCRIPTION
**Pull request by haukepetersen**
### Contribution description
Just minor doc fixes, should be quite straight forward:
- some doxygen blocks were not aligned to 4 spaces (this upset my text editor :-) )
- added missing information about return buffer sizes for some read functions

### Issues/PRs references
#7087

### CODE REVIEW

File: drivers/include/ata8520e.h [hunk 1]
```diff
@@ -175,24 +176,26 @@ void ata8520e_read_atmel_version(const ata8520e_t *dev, uint8_t *version);
...
+ *                          11 characters
  */
 void ata8520e_read_sigfox_version(const ata8520e_t *dev, char *version);
 
 /**
  * @brief Return the porting authorization code (PAC) available in the device
  *
  * @param[in]  dev          Pointer to device descriptor
- * @param[out] pac          String containing the pac
+ * @param[out] pac          String containing the pac, must be able to hold 8
```
Comment (aabadie) [inline drivers/include/ata8520e.h]: oups, ACK'ed to fast, this is not 8, but 16 here

File: drivers/include/ata8520e.h [hunk 2]
```diff
@@ -158,40 +158,44 @@ int ata8520e_init(ata8520e_t *dev, const ata8520e_params_t *params);
...
  */
 void ata8520e_read_sigfox_version(const ata8520e_t *dev, char *version);
 
 /**
  * @brief Return the porting authorization code (PAC) available in the device
  *
- * @param[in] dev                 Pointer to device descriptor
- * @param[out] pac                String containing the pac
+ * @param[in]  dev          Pointer to device descriptor
+ * @param[out] pac          String containing the pac, must be able to hold 8
```
Comment (aabadie) [inline drivers/include/ata8520e.h]: see here: https://github.com/RIOT-OS/RIOT/blob/master/drivers/include/ata8520e.h#L42

File: drivers/include/ata8520e.h (hunk 1 above)
Comment (haukepetersen) [reply drivers/include/ata8520e.h]: ups, I misread the code, just saw the `SIGFOX_PAC_LENGTH >> 1` but didn't notice that it was the parameter for `fmt_bytes_hex()` and not for `transfer_bytes`... Will fix.

Comment (haukepetersen) [reply drivers/include/ata8520e.h]: but wait, wouldn't it meant that the resulting string size is `17` then? The '_read_pac()` function copies 16 characters into the result string and then puts the `\0` byte in the 17th position (array offset 16) of the result string, right?

Comment (aabadie) [reply drivers/include/ata8520e.h]: You are right, I spoke too fast. That's what the driver test application does: see [here](https://github.com/RIOT-OS/RIOT/blob/master/tests/driver_ata8520e/main.c#L61)

Comment (haukepetersen) [reply drivers/include/ata8520e.h]: Perfect, then I will adapt this to the really correct numbers :-)

**Comment by haukepetersen** [comment]
adapted the required buffer sizes for `read_pac` and `read_id` (already squashed accidentally...)

File: drivers/include/ata8520e.h [hunk 3]
```diff
@@ -82,14 +82,14 @@ typedef enum {
...
 typedef enum {
-    ATA8520E_ATMEL_OK,                       /**< No error */
-    ATA8520E_ATMEL_COMMAND_ERROR,            /**< Command error / not supported */
-    ATA8520E_ATMEL_GENERIC_ERROR,            /**< Generic error */
-    ATA8520E_ATMEL_FREQUENCY_ERROR,          /**< Frequency error */
-    ATA8520E_ATMEL_USAGE_ERROR,              /**< Usage error */
-    ATA8520E_ATMEL_OPENING_ERROR,            /**< Opening error */
-    ATA8520E_ATMEL_CLOSING_ERROR,            /**< Closing error */
-    ATA8520E_ATMEL_SEND_ERROR                /**< Send error */
+    ATA8520E_ATMEL_OK,                      /**< No error */
```
Comment (aabadie) [inline drivers/include/ata8520e.h]: Please revert the doxygen 1 character shift of this enum, it seems unrelated.

Comment (haukepetersen) [reply drivers/include/ata8520e.h]: no, it is not, see the first commit of this PR: https://github.com/RIOT-OS/RIOT/pull/8449/commits/acac4290f5236c8bb16e267eb4813414e5b4de3e

Alignment in the master was off for one space for this block...

Comment (aabadie) [reply drivers/include/ata8520e.h]: > Alignment in the master was off for one space for this block...

I don't understand. Are you still talking of the "multiple of 4" indentations requirements? I don't consider this as a rule of thumb for alignment, thus the doxygen comment alignment is not 'off' in master. Maybe you should update your editor configuration ?

Comment (aabadie) [reply drivers/include/ata8520e.h]: I'm refering this comment: "some doxygen blocks were not aligned to 4 spaces (this upset my text editor :-) )"

Comment (haukepetersen) [reply drivers/include/ata8520e.h]: Yes, I am still talking about the 4-space alignment. In master, the this doxygen block is indented by 45 characters and `45 % 4 != 0`. So I put moved it to 44 characters -> now 4-space aligned.

> I don't consider this as a rule of thumb for alignment

But the coding convention does per default, as long as there is no good reason not to do so. I always interpreted this also applicable to doxygen blocks -> so yes, I say the 4-space alignment does apply here.

Furthermore, even if ignoring the 4-space alignment rule, it does not make sense to indent the block in questions here in a different style than the two blocks below...

Comment (aabadie) [reply drivers/include/ata8520e.h]: @haukepetersen what about stopping discussing what I'm requested and revert this change ? Those doxygen comments are aligned on their own and the code is nice enough (and not only because I wrote it ;) ). 

The only meaningful change of this PR is related to the documentation of payload sizes that are incorrect. Only this should be changed I think.

Comment (miri64) [reply drivers/include/ata8520e.h]: I think it is completely valid to piggy-back these kind of formatting fixes (and in this case it is a fix) with a related PR if the change in question is kept in its own commit. This is the case here, so let's not block this PR on a matter of opinion.

**Comment by haukepetersen** [comment]
removed those indention fixes. Will open a separate PR with those changes to have this discussion sorted out once and for all :-)

So this PR should be ready for merge, right?
